ServerlessLibraryUI/src/components/ConsentModal.vue

```diff
@@ -31,7 +31,7 @@
        <footer class="modal-footer">
           <slot name="footer">
             <div class="btn-agree">
-              <a :href="getDeployUrl(data)"  target="_blank" class="fullwidth-anchor" role="button" v-on:contextmenu="outboundRepoClick(data)" v-on:click="outboundRepoClick(data)" v-on:dblclick="outboundRepoClick(data)">
+              <a :href="getDeployUrl()"  target="_blank" class="fullwidth-anchor" role="button" v-on:click="outboundDeployClick()" v-on:dblclick="outboundDeployClick()">
```

Comment:
Are we leaving out contextmenu/right click on purpose ?

Reply:
yes. context menu has various options, and I could not find a way for us to know which item was eventually selected.